auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -231,6 +252,16 @@ private void showEmailAutoCompleteHint() {
         }
     }
 
+    private void checkEmailNotEmpty() {
```

Comment:
The library has it's own validation - for example you can't write invalid email. Use library's validation rules, don't create own logic for setting button color. If it's hard to do or it would require a lot of refactoring, don't change the color of the button at all and leave it always accent color.